docs/framework/guides/architecture/ui-components.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
UI components examples shouldn't be a part of the architecture category. It seems like architecture is describing the concepts behind the framework. This page is showcasing components. Let's move it one level up in hierarchy.